main/ertp/guide/purses-and-payments.md

```diff
@@ -1,109 +1,107 @@
 # Purses and Payments
- 
-Digital assets can be any of:
-- Currency-like, such as our imaginary Quatloos currency.
-- Goods-like digital assets, such as magic weapons for use in a game or theater tickets.
-- Other kinds of rights, such as the right to participate in a particular contract.
 
-In ERTP, digital assets exist in either a `purse` or a `payment`:
+There are different kinds of digital assets:
+- Currency-like, such as our imaginary Quatloos.
+- Goods-like, such as theater tickets or magic weapons for use in a game.
+- Abstract rights, such as authorization to participate in a particular contract.
```

Comment:
```suggestion
- Abstract rights, such as the right to participate in a particular contract.
```
Introducing the term "authorization" here seems like we might be introducing a new concept for the reader to learn, rather than just speaking informally. I think the original wording was fine.

Reply:
This was an attempt to fix the awkwardness of "rights, such as the right...". I've tweaked it to "Abstract rights, such as participation in a particular contract", but let me know if you still prefer the original and I'll revert.